src/assets/styles/main.scss

```diff
@@ -1,7 +1,138 @@
 html {
+  font-family: Freight, Avenir, Helvetica, Arial, sans-serif;
 }
 @import 'variables';
 @import 'nav';
 
+.home .site-header {
+  margin-bottom: 0 !important;
+}
+.wp-block-cover .wp-block-cover__inner-container p {
+  font-size: 2.875rem;
+  @media (max-width: $tabletL) {
+    font-size: 2.25rem !important;
+  }
+  @media (max-width: $mobileL) {
+    font-size: 1.875rem !important;
+    letter-spacing: 1 !important;
+  }
+}
+.product_title {
+  margin-top: 0 !important;
+}
+.entry-title {
+  font-family: Freight;
+  font-weight: $semiBold;
+  color: $pageHeaderText;
+  margin-top: -4.5%;
+  @media (max-width: $tabletL) {
+    margin-top: -8%;
+  }
+
+  @media (max-width: $mobileL) {
+    margin-top: -14%;
+    font-size: 2rem;
+  }
+}
+.page-selection {
+  margin-top: -3.5rem;
+  @media (max-width: $mobileL) {
+    margin-top: -3.1rem;
+  }
+}
+.page-id-107 .site-content,
```

Comment:
styling for specific page-id's can be tricky - incase we want to make a new product page or something (the id will change). It's nice instead to use page templates. For example we can make a page template for selection-page (copy paste page.php and rename to selection-template.php and add a string to the top --- check [this guide)](https://www.wpbeginner.com/wp-themes/how-to-create-a-custom-page-in-wordpress/). When you then pick this template in the wordpress admin (page-attributes) that page will get class like : template-selection.